spec/src/main/asciidoc/architecture.asciidoc

```diff
@@ -23,13 +23,67 @@
 
 The MicroProfile pom will identify the contents of each MicroProfile release.
 The contents will be those features and versions agreed to by the MicroProfile community.
-These features may be external to the MicroProfile community (Java EE features such as JAX-RS 2.1), or they be a component of the MicroProfile project (such as Config 1.3).
+These features may be external to the MicroProfile community (Jakarta EE features such as
+Jakarta RESTful Web Services 2.1),
+or they be a component of the MicroProfile project (such as Config 2.0).
 
 Since the MicroProfile repo is an "umbrella" project, there will be no individual API or TCK generated for this component.
 Only this Specification will be generated and maintained to document the contents of each release.
 
 In reverse chronological order, here's the specification for each MicroProfile release.
 
+[[microprofile4.0-M1]]
+=== MicroProfile 4.0 Milestone 1 (Q32020)
+// MicroProfile 4.0-M1 is the 14th platform release for the
+// https://projects.eclipse.org/projects/technology.microprofile[Eclipse MicroProfile project].
+MicroProfile 4.0-M1 is based on Jakarta EE 8, the first MicroProfile release to
+be based on Jakarta EE (replacing the role of Java EE).
+Although Jakarta EE 8 is API backwards compatible with Java EE 8, Jakarta EE replaces Java EE dependencies with
+Jakarta EE dependencies for all MicroProfile specifications, so all MicroProfile specifications
+are updated to major versions.
+
+The following specifications include API breaking changes:
```

Comment:
We agreed on the forum to say "incompatible changes" as API breaking changes does not cover all kinds of incompatible changes such as just behaviour changes.

Reply:
Will fix.